pkg/kubelet/cm/topologymanager/scope.go

```diff
@@ -68,7 +68,17 @@ func (s *scope) Name() string {
 	return s.name
 }
 
+func (s *scope) Lock() {
+	s.mutex.Lock()
+}
+
+func (s *scope) Unlock() {
+	s.mutex.Unlock()
+}
+
 func (s *scope) GetAffinity(podUID string, containerName string) TopologyHint {
+	s.mutex.Lock()
+	defer s.mutex.Unlock()
 	return s.podTopologyHints[podUID][containerName]
 }
```

Comment:
Following my advice above, this would then become:
```
func (s *scope) GetAffinity(podUID string, containerName string) TopologyHint {
	return s.getTopologyHints(podUID, containerName)
}
```